src/features/collapsible-sidebar/CollapsibleSidebar.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @flow
+ * @file Sidebar component that supports rendering different elements based on expand/collapse state
+ * @author Box
+ *
+ * A sidebar component that supports collapsed/expanded state and responsive sizing.
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+import styled from 'styled-components';
+
+import './CollapsibleSidebar.scss';
+
+const StyledNav = styled.nav`
+    background-color: ${props => props.theme.primary.background};
+    border-right: 1px solid ${props => props.theme.primary.border};
+    color: ${props => props.theme.primary.foreground};
+
+    .crawler > div {
+        background-color: ${props => props.theme.primary.foreground};
+    }
```

Comment:
In Admin Console, the left nav's background color will not be of primary color. It will be #222 for every admin user in any enterprise. Said that, can the color assignment be made more generic?

Reply:
For that use case, you would create a new theme with `#222` as the primary color